lib/features/login/data/network/dns_lookup/dns_lookup_priority.dart

```diff
@@ -0,0 +1,27 @@
+/// Represents the priority order and description of different DNS lookup modes.
+///
+/// Priority level increases with fallback order:
+/// 1 → System default
+/// 2 → Public resolvers (UDP/TCP or DOH)
+/// 3 → Cloud resolvers (Google/Cloudflare)
+enum DnsLookupPriority {
+  /// Uses the device's system-configured DNS (e.g., from ISP or OS settings).
+  system(1, 'System Default'),
+
+  /// Uses open DNS resolvers accessible via UDP/TCP (e.g., Quad9, OpenDNS).
+  publicUdp(2, 'Public DNS (UDP/TCP)'),
+
+  /// Uses DNS-over-HTTPS (DoH) resolvers for secure name resolution.
+  publicDoh(2, 'Public DNS (DoH)'),
```

Comment:
why it still 2?

Reply:
Both use Public DNS just different implementation so it has the same priority